src/main/java/de/uhd/ifi/se/decision/management/jira/quality/MetricCalculator.java

```diff
@@ -420,47 +414,55 @@ public MetricCalculator(Long projectId, ApplicationUser user, String issueTypeId
 			 */
 
 		}
-		numberOfRelevantSentences.put("Relevant Sentences", isRelevant);
-		numberOfRelevantSentences.put("Irrelevant Sentences", isIrrelevant);
+		numberOfRelevantSentences.put("Relevant Comment", isRelevant);
+		numberOfRelevantSentences.put("Irrelevant Comment", isIrrelevant);
 		return numberOfRelevantSentences;
 	}
 
-	public Map<String, String> getLinksToIssueTypeMap(KnowledgeType knowledgeType) {
-		LOGGER.info("RequirementsDashboard getLinksToIssueTypeMap");
-		if (knowledgeType == null) {
-			return null;
-		}
+	public Map<String, String> getLinksToIssueTypeMap(KnowledgeType knowledgeType, int linkDistance) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'getLinksToIssueTypeMap(KnowledgeType, int)' has an NPath complexity of 297](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5441305)